snuba/settings.py

```diff
@@ -81,8 +82,12 @@
     "bootstrap.servers": os.environ.get("DEFAULT_BROKERS", "localhost:9092"),
 }
 STORAGE_BROKER_CONFIG: Mapping[str, Mapping[str, Any]] = {}
```

Comment:
I think this change should have an impact on this config as well. 
If we are abstracting the topic config, a storage has a stream associated, and a stream correspond to one and only one topic, then the topic should be connected to the broker and not the storage as a topic is univocally associated to the broker, not the storage.